Makefile

```diff
@@ -26,7 +27,8 @@ drafter: config.gypi $(BUILD_DIR)/Makefile
 	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/$@ ./bin/$@
 
 install: drafter
-	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(DESTDIR)/drafter
+	mkdir -p $(BINDIR)
+	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(BINDIR)/drafter
```

Comment:
Also, this install phase doesn't install the shared library or headers. Is there some reason it hasn't done this in the past @pksunkara?

I'd expect it would install the library so that I can build C or C++ tools/libraries that depend on it.